Source/Core/CMakeLists.txt

```diff
@@ -2,7 +2,10 @@ add_subdirectory(AudioCommon)
 add_subdirectory(Common)
 add_subdirectory(Core)
 add_subdirectory(DiscIO)
-add_subdirectory(DolphinWX)
+if(NOT DISABLE_WX)
```

Comment:
I don't believe we need this here as we could check it in the file itself, but it's fine for now until we clean everything up.